ass-03-c.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/*ID: 602110195
+Name: Zhang Hao(Henry)
+Wechat: hikki*/
+$c=fopen($_SERVER['argv'][2],'r');
+for($k=0;;$k++){
+    $cg[$k]=trim(fgets($c,4096));
+    if(feof($c)){
+    break;
+    }
+}fclose($c);
+$cga=implode(" ",$cg);
+$cgs=explode(" ",$cga);

```

Comment:
If you use this approach, not only it changes the group of words but also change the single word, e.g. university, that we don't want.